pkg/operator/targetconfigcontroller/targetconfigcontroller.go

```diff
@@ -327,6 +336,15 @@ func managePod_v311_00_to_latest(ctx context.Context, configMapsGetter corev1cli
 		required.Spec.Containers[0].Args = append(required.Spec.Containers[0].Args, "--tls-private-key-file=/etc/kubernetes/static-pod-resources/secrets/serving-cert/tls.key")
 	}
 
+	config, err := configInformer.Config().V1().Schedulers().Lister().Get("cluster")
```

Comment:
This is changing, see https://github.com/openshift/cluster-kube-scheduler-operator/pull/254

Reply:
@soltysh is this change just moving from a slice of args to a concatenated string?

Reply:
Yup